### DISCUSSION TITLE
Is there an "official" coding style for phoebus?

### DISCUSSION THREAD

**Discussion by lorenzo-gomez-windhover** [ControlSystemStudio/phoebus#2098] [General]
Hi there,

hope you are all doing well.

Just asking because one thing one could do is add the `fmt-maven-plugin` plugin to the maven project. And this would allow us to keep the coding style consistent across the entire codebase. It has different goals to check the source code if it is formatted properly accoding to [google coding style](https://google.github.io/styleguide/javaguide.html) and to also enforce the code style if one's coding style is not properly formatted before committing/pushing.

**Comment by kasemir** [comment]
See https://github.com/ControlSystemStudio/phoebus/issues/894 and https://github.com/ControlSystemStudio/phoebus/pull/1031/files
